### DISCUSSION TITLE
Host and serve the saved BERTopic Model

### DISCUSSION THREAD

**Discussion by yugkha3** [MaartenGr/BERTopic#1849] [Q&A]
What cloud services, packages, workflows y'all use to host and serve the saved models? Very eager to know!!

**Comment by Btibert3** [comment]
The docs do a good job identifying some of the considerations for serializing a model, which in part would drive your solution.  Your question is timely because I am working on a project that will require deploying a model trained on an initial corpus for inspection of new open-ended comments.  I will follow up here on some of my learnings unless someone else beats me to it.

**Reply by Ciaran0** [reply]
What path did you take? Would love to know!
